src/main/java/com/example/online_shop_api/Dto/Response/ProductResponseDto.java

```diff
@@ -1,14 +1,22 @@
 package com.example.online_shop_api.Dto.Response;
 
-import lombok.Data;
-
+import com.example.online_shop_api.Entity.ProductHelpers.Brand;
+import com.example.online_shop_api.Entity.ProductHelpers.Color;
 import java.math.BigDecimal;
 
+import lombok.*;
+
+@Builder
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class ProductResponseDto {
     private Long id;
     private String name;
+    private String productType;
     private BigDecimal price;
     private int quantity;
     private String imageLocation;
```

Comment:
Is this not supposed to be List<String> imageLocations?

Reply:
this field is added to understand when retrieving a given product its type, we already have  **_String imageLocation;_**

Reply:
Won't we have a list of images, not just one image and its location? Or we will add just one when creating a new product? We can discuss this.